frontend/tests/builders/index.ts

```diff
@@ -0,0 +1,5 @@
+import createLineItem from './createLineItem'
+import createOffer from './createOffer'
+import createPallet from './createPallet'
+
+export { createLineItem, createOffer, createPallet }
```

Comment:
**nitpick:** maintaining a barrel file is extra effort and does IMO no significant advantages. Consider removing it.